geomesa-tools/bin/geomesa-kafka

```diff
@@ -0,0 +1,251 @@
+#! /usr/bin/env bash
+#
+# Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0 which
+# accompanies this distribution and is available at
+# http://www.opensource.org/licenses/apache2.0.php.
+#
+
+setGeoHome ()
```

Comment:
it would be nice if we had a common script that the other two (geomesa-accumulo and geomesa-kafka) could `source` to pick up the common functions and code.